### ISSUE TITLE
output of running fourmolu sometimes doesn't appear in the GitHub Action logs

### ISSUE THREAD

**Issue by cdepillabout** [haskell-actions/run-fourmolu#4]
> It is incredibly difficult today to determine where the fault lies in the codebase, since the Action doesn't say anything about which files are not formatted, neither in the files view, nor in the action interface 
>
> ![Screenshot_20220611_223009](https://user-images.githubusercontent.com/15848443/173203911-5584c971-a407-4a6b-9f59-1de6803fc054.png)
>
> This is the [faulty run](https://github.com/flora-pm/flora-server/runs/6845560053?check_suite_focus=true)

_Originally posted by @tchoutri in https://github.com/fourmolu/fourmolu-action/issues/1#issuecomment-1152993174_

-----------------------------------------------------------------

As shown above in https://github.com/flora-pm/flora-server/runs/6845560053?check_suite_focus=true, it does appear that this fourmolu action is not correctly outputting which files are not formatted correctly.

**Comment by cdepillabout** [comment]
I created a test PR on this repo just to make sure that the fourmolu-action does correctly give warnings in #5.  It appears to work as expected.

@tchoutri I took a look at your job, and the one suspicious thing I noticed is that you aren't calling the checkout action anywhere:

https://github.com/flora-pm/flora-server/blob/45d3c8a0fabf540e0b8086a12c354b3358bc5620/.github/workflows/linting.yml#L9-L17

You'll likely need the checkout action as well just so the fourmolu action can see your Haskell source code?

https://github.com/flora-pm/flora-server/blob/45d3c8a0fabf540e0b8086a12c354b3358bc5620/.github/workflows/linting.yml#L22

I imagine we need a "full" example in the README that can easily be copy-pasted.
